### PR TITLE
Bratseth/nn query type resolving

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

File: config-model/src/test/java/com/yahoo/searchdefinition/derived/NearestNeighborTestCase.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (geirst) [inline config-model/src/test/java/com/yahoo/searchdefinition/derived/NearestNeighborTestCase.java]: Please use updated copyright.

**Review by geirst — APPROVED** [review]
👍

File: config-model/src/test/java/com/yahoo/searchdefinition/derived/NearestNeighborTestCase.java (hunk 1 above)
Comment (bratseth) [reply config-model/src/test/java/com/yahoo/searchdefinition/derived/NearestNeighborTestCase.java]: We don't need to update copyright when we change a file.

Comment (geirst) [reply config-model/src/test/java/com/yahoo/searchdefinition/derived/NearestNeighborTestCase.java]: Yes, but this is a new file.

Comment (bratseth) [reply config-model/src/test/java/com/yahoo/searchdefinition/derived/NearestNeighborTestCase.java]: Ah, you're right

**Pull request merged**